README.md

```diff
@@ -1,92 +1,60 @@
-- In VSCode, press `CMD` + `SHIFT` + `P` to open the command prompt
-- Search for and select the command `Preferences: Open Settings (UI)`
-- Choose either the "User" or "Workspace" tab
+<!-- TODO: Link 'HubSpot Developer Docs' text to CMS Dev Docs url when they are live -->
+To read more about this extension and its features, please see the documentation at the HubSpot Developer Docs. If you're new to CMS Hub, check out [how to get started with local development](https://designers.hubspot.com/docs/tools/local-development).
+
+## Settings
+
+### Language Modes
+
+This extension introduces new file languages: `HTML + HUBL` and `CSS + HUBL`. In order to use the full set of features from the extension, VS Code needs to be aware of these languages and how they are associated with various file types. For one-off files, you can simply change the Language Mode in the lower right hand status bar of the editor. However, we recommend adjusting your file associations under your User or Workspace preferences so HubL syntax highlighting will automatically be applied to all of your projects:
+
+- In VSCode, press `CMD + SHIFT + P` to open the command prompt
+- Search for and select the command `Preferences: Open User Settings`
+- Choose either the "User" or "Workspace" tab to apply these settings
 - In "Search settings" look up `files.associations`
 - Select "Add Item" and add these file associations: `*.html: html-hubl` and `*.css: css-hubl`
+- For more information about how VSCode settings work, [please read the official VS Code documentation](https://code.visualstudio.com/docs/getstarted/settings).
+
+### Enabling Emmet for HTML/HubL files
+
+To enable Emmet on your `html-hubl` files, you can map `html-hubl` to `html` in your settings under "Emmet: Include Languages"
+
+![VS Code Setting for Emmet](https://user-images.githubusercontent.com/9009552/114593899-9e320500-9c5a-11eb-98c6-9de022344ebc.png)
+
+### IntelliSense Suggestions
+
+If you are would like to get IntelliSense suggestions when in snippet placeholders, you will need to add the following to your user settings:
+
+`"editor.suggest.snippetsPreventQuickSuggestions": false`
+
+For parameter suggestions, the following should also be added:
+
+`"editor.parameterHints": true`
+
+## Beta Features
+
+To enable beta features, navigate to `HubSpot > Beta` in your VS Code settings and check the box to opt in. Any current beta features are listed below.
+
+### Current list of features available in Beta:
+
+None at this time.
+
+## Telemetry
+
+HubSpot for VS Code collects user data in order to improve the extension’s experience. You can [review HubSpot’s privacy policy here](https://legal.hubspot.com/privacy-policy). Additionally, you may opt out of data collection by changing the setting for global telemetry in VS Code. To read more about VS Code and telemetry, including disabling telemetry reporting, [please read the official VS Code documentation](https://code.visualstudio.com/docs/getstarted/telemetry).
```

Comment:
Is it worth mentioning that we also respect the CLI's tracking preferences?

Reply:
@miketalley - I might be mistaken here on where we ended up with tracking and agreeing to it in the extension -- Are we using the CLI's tracking preferences, the telemetry setting in VS Code, or agreement in both areas in order to track users?

Reply:
@miketalley feel free to weigh in but I think it's probably ok to keep as is IF we do respect both, meaning a "no" in the CLI or VS Code.

At that point, it doesn't really matter to the user as long as they can turn it off directly in VS Code

Reply:
We currently respect both the `allowUsageTracking` boolean setting in the `hubspot.config.yml` as well as the VSCode global telemetry setting (`workspace.getConfiguration().telemetry.enableTelemetry`).

I think respecting the CLI flag is a bonus that we don't _need_ to mention here. I don't see a case where someone is confused as to why their data is not being tracked because the `allowUsageTracking: false` exists in the `hubspot.config.yml`, since this needs to be set manually through editing the config or `hs config set`.